prov/efa/src/efa_domain.c

```diff
@@ -206,9 +207,10 @@ int efa_domain_open(struct fid_fabric *fabric_fid, struct fi_info *info,
 {
 	struct efa_domain *domain;
 	struct efa_fabric *fabric;
+	struct rxr_domain *rxr_domain;
```

Comment:
Why declare this at top of function scope, when it is only used in a single code block?